src/Compilers/Server/VBCSCompiler/DesktopBuildServerController.cs

```diff
@@ -103,24 +103,16 @@ protected override int RunServerCore(string pipeName, IClientConnectionHost conn
             // pipename and consuming excess resources. If someone else holds the mutex
             // exit immediately with a non-zero exit code
             var mutexName = BuildServerConnection.GetServerMutexName(pipeName);
-            bool holdsMutex;
-            using (var serverMutex = new Mutex(initiallyOwned: true,
-                                               name: mutexName,
-                                               createdNew: out holdsMutex))
+            bool createdNew;
```

Comment:
Would keep this name as `holdsMutex` as it's more instructive of what is going on here

Reply:
Didn't you ask me to change it? Should I make the var and the parameter both holdsMutex? I can do that